docs/developer-guide/code-contributions.md

```diff
@@ -15,8 +15,7 @@ first.
 
 Each enhancement proposal needs to go through our
 [triage process](#triage-process)
```

Comment:
TBH I wonder about that whole triage process thing, since I've never seen this process applied in practice 🤔

Reply:
Hmmm, what do you suggest? Do you think I should remove it as well?

Reply:
Yeah, let's probably remove this.

Reply:
Done.